documentation/how-to/digitize.en.md

```diff
@@ -139,6 +139,28 @@ A video demonstration of some editing possibilities:
 
 ![type:video](https://player.vimeo.com/video/499564854)<!-- markdown-link-check-disable-line -->
 
+## Erase Tool Functionality
```

Comment:
This should be moved to sit alongside the other tool, have it be a 3rd-level header, and harmonize title with the other tools (i.e. Erase tool, not Erase Tool Functionality).

Reply:
What do you think if it moved to the same section of reshape tool?